mtags/src/main/scala-3/scala/meta/internal/pc/AutoImports.scala

```diff
@@ -202,6 +220,41 @@ object AutoImports:
             case _ => None
         case IndexedContext.Result.InScope => None
 
+    def inferSymbolImport(symbol: Symbol): SymbolImport =
```

Comment:
This PR adds some duplications. `SymbolImport` is an extended form of `AutoImport` that allows to customize import section edits logic.

Previously, we were adding only one import per action but for exhaustive match completions we need to add several at once.
I'm going to get rid of `AutoImport` in other places but I think it would be better to merge it as it is as this change blocks @jkciesluk